apps/web/components/email-list/EmailList.tsx

```diff
@@ -33,13 +33,24 @@ import {
 } from "@/providers/QueueProvider";
 import { selectedEmailAtom } from "@/store/email";
 import { categorizeAction } from "@/utils/actions/categorize";
+import { Button, ButtonLoader } from "@/components/ui/button";
+import { ChevronDown } from "lucide-react";
 
 export function List(props: {
   emails: Thread[];
   type?: string;
   refetch: (removedThreadIds?: string[]) => void;
+  isLoadMore?: boolean;
```

Comment:
Could name better. eg `showLoadMore`. AI is good at helping with better naming.

Reply:
Done